hs_core/templatetags/hydroshare_tags.py

```diff
@@ -82,7 +97,11 @@ def resource_type(content):
 def resource_first_author(content):
     if not content:
         return ''
-    first_creator = content.first_creator
+    first_creator = None
+    for creator in content.metadata.creators.all():
+        if creator.order == 1:
+            first_creator = creator
+            break
     if first_creator.name and first_creator.relative_uri:
```

Comment:
During testing on dev-2, I found this bug. It seems first_creator can be None. So before accessing first_creator.name need to check the creator is not None.

Reply:
good catch, thanks